src/Coin.cpp

```diff
@@ -278,6 +278,14 @@ Hash::Hasher TW::base58Hasher(TWCoinType coin) {
     return getCoinInfo(coin).base58Hasher;
 }
 
+uint32_t TW::slip44Id(TWCoinType coin) {
+    auto id = getCoinInfo(coin).numericId;
+    switch (coin) {
+    case TWCoinTypeBinanceSmartChain: return id / 10000;
```

Comment:
I prefer a predefined range, like id + 10000000, 10000714, instead of id * 10000.

Reply:
See my comment above, I'd like to get it optionally from `CoinInfoData`, is that any good way to achieve this? so we can remove hacky code here

Reply:
OK, but the numericalId of BSC is still present (not at this point of the code, sorry).  So this comment applies to numericalId of BSC.

Reply:
I see you already changed it